indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -388,6 +388,8 @@
             :metadata-format (name (mt/format-key format))
             :related-urls (map json/generate-string opendata-related-urls)
             :has-opendap-url (not (empty? (filter opendap-util/opendap-url? related-urls)))
+            :cloud-hosted (or (not (empty? s3-bucket-and-object-prefix-names))
```

Comment:
you can't use s3-bucket-and-object-prefix-names - that is an optional element. You should check to see if DirectDistributionInformation exists.

Reply:
yea, good catch, this was just wrong.